data/tools/abaplint.yml

```diff
@@ -0,0 +1,10 @@
+name: abaplint
+category:
```

Comment:
Think it should be `categories` to reflect that it's a list. Otherwise I'd assume a single value as a first-time contributor.